vllm/attention/backends/ipex_attn.py

```diff
@@ -214,6 +214,12 @@ def use_sdp_causal(head_dim, query_states):
         and query_states.dtype in [torch.float, torch.half]     # fp32/fp16
     )
 
+def use_gqa_kernel(num_heads, num_kv_heads):
+    kv_cache_format = os.environ.get('USE_VLLM_KVCACHE')
```

Comment:
Can we remove this env and just check kv_heads?

Reply:
It may be easier to check correctness and performance between `paged_attention_gqa()` and `paged_attention()` with a env setting? Would remove this after the kernel is verified with different models.